var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -72,6 +72,7 @@ class Ucx(AutotoolsPackage, CudaPackage):
         description="Build shared libs, static libs or both",
     )
     variant("logging", default=False, description="Enable logging")
+    variant("numa", default=False, description="Enable NUMA support")
```

Comment:
```suggestion
    variant("numa", default=False, when="@:1.14", description="Enable NUMA support")
```

Reply:
@haampie Would you help explain version ranges like this? I see it everywhere in packages but hard to understand what does it mean actually.

Reply:
So what I understood it will apply for all the versions `<= 1.14.0`

Reply:
Also let's not add `when` condition for now.

Reply:
See https://spack.readthedocs.io/en/latest/basic_usage.html#version-specifier

`@:1.14` includes 1.14.x but excludes 1.15 and above